scene/gui/item_list.cpp

```diff
@@ -942,6 +942,7 @@ void ItemList::_notification(int p_what) {
 				}
 			}
 
+			minimum_size_changed();
```

Comment:
FYI, I'm adding this because ItemList don't update minimum size when adding/modifying/etc and I don't want a regression in the about dialog.